secure-biometric-interface-integrator/lib/secureBiometricInterface.js

```diff
@@ -548,14 +590,29 @@ class SecureBiometricInterface {
     }
   }
 
+  /**
+   * Cancel loading indicator
+   */
+  cancelLoadingIndicator() {
+    this.discoveryCancellationFlag = true;
+    this.errorStateChanged(
+      {
+        errorCode: ErrorCode.DEVICE_NOT_FOUND,
+        defaultMsg: "Device not found",
+      },
+      false
+    );
+    this.statusChanged(states.LOADED);
```

Comment:
Why is the status changed to loaded?

Reply:
It actually denote that the component is currently not in a LOADING or AUTHENTICATING state, thats why we used LOADED

Reply:
@gsasikumar , LOADED is a state representing no asynchronous call.